lib/packwerk/package_set.rb

```diff
@@ -28,8 +28,12 @@ def load_all_from(root_path, package_pathspec: nil)
       def package_paths(root_path, package_pathspec)
         bundle_path_match = Bundler.bundle_path.join("**").to_s
 
-        Dir.glob(File.join(root_path, package_pathspec, PACKAGE_CONFIG_FILENAME))
-          .map { |path| Pathname.new(path) }.reject { |path| path.realpath.fnmatch(bundle_path_match) }
+        glob_patterns = Array(package_pathspec).map do |pathspec|
+          File.join(root_path, pathspec, PACKAGE_CONFIG_FILENAME)
+        end
+
+        Dir.glob(glob_patterns)
+          .map { |path| Pathname.new(path).cleanpath }.reject { |path| path.realpath.fnmatch(bundle_path_match) }
```

Comment:
`.cleanpath` normalizes the glob pattern to convert things like `"packwerk_app/./app/sales"` into `"packwerk_app/app/sales"`.

Reply:
\nit can we multiline the chained methods to make it easier to read?